Example/Example/Authors/AuthorsView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  AuthorsView.swift
+//  Example
+//
+//  Created by Martin MOIZARD-LANVIN on 15/09/16.
+//  Copyright © 2016 Akane. All rights reserved.
+//
+
+import Foundation
+import Akane
+
+class AuthorsView : UITableView, ComponentView {
+   override func awakeFromNib() {
+      self.estimatedRowHeight = 44;
```

Comment:
You can also use a `TableViewFlowLayout`, but this is for more complex cases. This is perfectly fine here ;)